examples/eSealing/src/eSealing.tsx

```diff
@@ -0,0 +1,413 @@
+                                onChange={() => {
+                                    setIsRegisterFilePage(!isRegisterFilePage);
+                                    setFileHash('');
+                                    setFileHashHex('');
+                                    setWitness('');
+                                    setTimestamp('');
+                                    setHash('');
+                                }}
+                                onColor="#979797"
+                                offColor="#979797"
+                                onHandleColor="#308274"
+                                offHandleColor="#308274"
+                                checked={!isRegisterFilePage}
+                                checkedIcon={false}
+                                uncheckedIcon={false}
+                            />
+                            <button
+                                style={
+                                    activeConnectedAccount === undefined || isRegisterFilePage === true
+                                        ? ButtonStyleDisabled
+                                        : ButtonStyle
+                                }
+                                type="button"
+                                disabled={activeConnectedAccount === undefined}
+                                onClick={() => {
+                                    setIsRegisterFilePage(false);
+                                    setFileHash('');
+                                    setFileHashHex('');
+                                    setWitness('');
+                                    setTimestamp('');
+                                    setHash('');
+                                }}
+                            >
+                                Display File Hash Record
+                            </button>
+                        </div>
+                    </>
+                )}
+                {activeConnectionGenesisHash && activeConnectionGenesisHash !== network.genesisHash && (
+                    <p style={{ color: 'red' }}>
+                        Unexpected genesis hash: Please ensure that your wallet is connected to network{' '}
+                        <code>{network.name}</code>.
+                    </p>
+                )}
+            </div>
+
+            <label>
+                {activeConnectedAccount !== undefined && (
+                    <>
+                        <p style={{ marginBottom: 0 }}>Upload a file:</p>
+                        <input
+                            className="input"
+                            style={InputFieldStyle}
+                            type="file"
+                            onChange={changeHandler}
+                            ref={file}
+                        />
+                        <button
+                            style={ButtonStyle}
+                            type="button"
+                            onClick={async () => {
```

Comment:
I think you should use an explicit promise here as otherwise any exception will not get caught. Maybe surrounding with try/catch is sufficient.